src/stylesheets/elements/_buttons.scss

```diff
@@ -1,181 +1,215 @@
 .usa-button {
-  @include u-margin-y("1px");
```

Comment:
Make sure to check in IE, Edge, Firefox, Safari, and Chrome/accessibility testing as discussed in standup